security/tpm/src/Microsoft.Azure.Devices.Provisioning.Security.Tpm.csproj

```diff
@@ -28,7 +28,7 @@
   </ItemGroup>
 
   <PropertyGroup>
-    <Version>1.13.1</Version>
```

Comment:
I assume since this is the first time we are releasing a preview from this branch since it was created off of master, we never changed these versions and that's why they are all referencing our GA versions?

Reply:
Yupp, I've mentioned in my comments :)

Reply:
Ah, I see ... at the top of the release notes ...